scripts/sources.conf

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Do not edit this file, create sources.local.conf for overriding.
+#
+
+# Some random port because 8000 is typically already used.
+export PORT=8131
+export DATABASE_HOST=localhost
+export DATABASE_PORT=5432
+export DATABASE_NAME=sources_devel
+export DATABASE_USER=sources_user
+# Local users are considered trusted via pg_hba.conf but
+# password must be present otherwise sources won'b boot.
+export DATABASE_PASSWORD=trusted
+export CACHE_HOST=localhost
+export CACHE_PORT=6379
+export CACHE_PASSWORD=no_authentication
```

Comment:
I do not think these are used :thinking: I was looking for them in their code and it is not there...

Reply:
@adiabramovitch can you comment? You had it there so perhaps there is a reason.

Reply:
I don't recall, If you didn't need those, maybe they are redundant.

Reply:
Ok let's drop it then.